docs/insights/coverage.md

```diff
@@ -0,0 +1,63 @@
+---
+id: coverage
+title: Measuring Your Test Coverage
+sidebar_label: Coverage
+description: See a breakdown of the browser and device variations covered by your tests.
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The Coverage report allows Org Admins to view and download the breakdown of browsers and mobile devices the tests across their organization have run against.
+
+:::note Org Admins Only
+You must have [Org Admin](/basics/acct-team-mgmt/managing-user-info/#user-roles) privileges for your organization in order to view and download coverage data.
+:::
+
+## Access the Coverage Reports
+
+1. Click the **Insights** tab to expand its submenu.
+1. Click **Coverage** (only visible to Org Admin users).
+1. Specify the type of coverage you wish to view by selecting the **Devices** or **Browsers** tab.
+1. Specify whether you wish to see coverage data for **Virtual Cloud** tests (desktop web-apps, mobile emulators and simulators) or **Real Devices** (physical devices located in our data centers and connected to our platform).
+1.Use the [filters](scope.md#using-filters-to-adjust-the-scope-of-your-data) to show results for a particular user's tests and/or a specific time period.
+
+Once you have completed your selections, the report shows a breakdown of all tests that have been run for the specified user during the specified time period, providing the exact number and total duration of tests run in each environment with a bar graph for easy visualization.
+
+<img src={useBaseUrl('img/insights/coverage.png')} alt="Coverage Report" width="900"/>
+
+:::note Zero Coverage Data
+The coverage reports are not intended to identify test environments for which you have no coverage. That is to say, the report does not show the set of browsers or devices that are supported by Sauce Labs but for which the number of sessions run is zero.
```

Comment:
I don't understand this paragraph.
Can you elaborate what is meant by "identify test environments"?
What does "but for which the number of sessions run is zero"